src/manager.rs

```diff
@@ -244,4 +245,46 @@ mod tests {
             .expect("success");
         assert!(!Arc::ptr_eq(&path2_arc, &arc));
     }
+
+    /// Test that one can mutate managed Rkv instances in surprising ways.
```

Comment:
Might be nice to mention https://github.com/mozilla/rkv/pull/130 (or a corresponding issue, but I don't see one)

This doesn't seem to be directly related to the issue, did you somehow suffered from this behavior while writing the patch? 🙂

Reply:
Wow, it's been some time :D -- I think so? I don't remember how I came to write this.

Reply:
Ok, I really have no idea what exactly I wanted to show with this test. Maybe that's indeed #130? 
Let me take this commit out of this PR though.